pkg/locodedb/calls.go

```diff
@@ -0,0 +1,55 @@
+package locodedb
+
+import (
+	"errors"
+)
+
+// ErrRecordNotFound is returned when the record is not found in the location database.
+var ErrRecordNotFound = errors.New("record not found")
+
+var (
+	// mCountries is a map of country codes to country names.
+	mCountries map[CountryCode]string
+
+	// mLocodes is a map of location codes to location records. The location code is a concatenation of the country code and the location code. The Record contains country name, the location name, the subdivision name, the subdivision code, the Point and the Continent.
```

Comment:
this line is extremely long i would say. 80 chars long is smth we should strive for